src/ts/component/menu/dataview/view/settings.tsx

```diff
@@ -307,6 +320,8 @@ const MenuViewSettings = observer(class MenuViewSettings extends React.Component
 				caption: allowedDefaultType ? defaultTypeName : this.defaultTemplateName,
 				data: {
 					typeId,
+					hasSources,
```

Comment:
make hasSources check the same as in this menu, do not pass it as a param inside inner menu